### ISSUE TITLE
Missing yargs dependency

### ISSUE THREAD

**Issue by dashed** [mswjs/msw#11]
I tried running `yarn msw create public/`, which resulted in an error:

```
TypeError: yargs.positional is not a function
    at Object.yargs.usage.command [as builder] (/path/to/node_modules/msw/cli/msw.js:10:13)
    at Object.self.runCommand (/path/to/node_modules/yargs/lib/command.js:193:35)
    at Object.Yargs.self._parseArgs (/path/to/node_modules/yargs/yargs.js:990:30)
    at Object.get [as argv] (/path/to/node_modules/yargs/yargs.js:930:19)
    at Object.<anonymous> (/path/to/node_modules/msw/cli/msw.js:18:10)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
error Command failed with exit code 1.
```

It looks like `yargs` is missing as a dep from `package.json` in this commit: https://github.com/kettanaito/msw/commit/a8a9854114caa353f6ca45c6a9acdb232f4d0d75

**Comment by kettanaito** [comment]
Hello, @dashed. Thank you for reporting the issue.

Indeed, I think `yargs` must be listed as a save dependency, not a dev dependency of the library.
I will provide the fix and publish in the next patch version.

**Comment by kettanaito** [comment]
I've just published `msw@0.2.1` with `yargs` properly included as a save dependency.

Could you please update and let me know if the issue is resolved?
Following the [CRA example](https://github.com/kettanaito/msw/tree/master/example) might be helpful.

**Comment by dashed** [comment]
Seems to work fine. Thanks! 👍
